pallets/dactr/Cargo.toml

```diff
@@ -12,29 +12,33 @@ description = "Data Avail Control Module"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-avail-core = { git = "https://github.com/availproject/avail-core", tag = "node-v1.9.0.0", default-features = false, features = ["runtime"] }
-kate = { git = "https://github.com/availproject/avail-core", tag = "node-v1.9.0.0", default-features = false }
+avail-core = { workspace = true, default-features = false }
+kate = { workspace = true, default-features = false }
+frame-system = { path = "../../pallets/system", default-features = false }
+pallet-transaction-payment = { path = "../../pallets/transaction-payment", default-features = false }
 
-log = { version = "0.4.17", default-features = false }
+log = { version = "0.4.20", default-features = false }
 static_assertions = "1.1.0"
 
 # Substrate
 serde = { version = "1.0.126", optional = true, features = ["derive"] }
```

Comment:
Let's check if all our serde versions are the same. I see that for avail-subxt we use `1.0.195` and here we use `1.0.126`